pandas/plotting/_core.py

```diff
@@ -2874,25 +2874,62 @@ def scatter(self, x, y, s=None, c=None, **kwds):
     def hexbin(self, x, y, C=None, reduce_C_function=None, gridsize=None,
                **kwds):
         """
-        Hexbin plot
+        Make hexagonal binning plots.
+
+        Make an hexagonal binning plot of `x` versus `y`, where `x`,
+        `y` are 1-D sequences of the same length, `N`. If `C` is `None`
+        (the default), this is an histogram of the number of occurrences
```

Comment:
lgtm. if you can find a referene on wikipedia might be nice to link.

Reply:
I'm afraid there is no reference in wikipedia to hexagonal binning. The closed topic  I found in wiki is [the data binning article](https://en.wikipedia.org/wiki/Data_binning). Maybe it's a little bit to generic to include it in the hexbin docstring since it is also suitable for histogram and histogram2d